lib/signup/sign_up_screen.dart

```diff
@@ -105,11 +137,14 @@ class _HomeContent extends StatelessWidget {
                   // ),
                   BlocConsumer<SignUpCubit, SignUpState>(
                     listener: (context, state) {
+                      if (kDebugMode) {
+                           print('$state');
+                         }
                       if(state is SignUpFailedState && state.provider == 'Github')
```

Comment:
why don't u create different state for GitHub, since behaviour is different?

Reply:
Then we should create specific state for each login like Github/Google

Reply:
Revised the code and checked in